doc/source/index.rst

```diff
@@ -64,20 +64,33 @@ Documentation Index
 
 .. toctree::
    :maxdepth: 1
-   :caption: Language Wrappers (Production)
```

Comment:
I think we should add a main section to index for batch? I added a streaming section to the kafka PR.

Reply:
Oh right, ok makes sense

Reply:
Done